src/lib.rs

```diff
@@ -115,26 +109,52 @@ const LIB_MSVC: &str = "lib.exe";
 ///
 /// The compile target environment ABI name (as in `CARGO_CFG_TARGET_ENV`)
 /// is passed in `env`.
-pub fn generate_implib_for_target(out_dir: &Path, arch: &str, env: &str) -> Result<()> {
+pub fn generate_implib_for_target(
+    version: Option<(u8, u8)>,
+    out_dir: &Path,
+    arch: &str,
+    env: &str,
+) -> Result<()> {
```

Comment:
This is a breaking API change requiring a major crate version bump.

Maybe we could add a new public function instead with a signature like `generate_versioned_implib_for_target(out_dir: &Path, version: (u8, u8), ...)`?

Reply:
Perhaps this should be a builder?

Reply:
> Perhaps this should be a builder?

This is a good idea for the future `v0.3` API, but I'm not sure we need build it just for this feature.

Reply:
> > Perhaps this should be a builder?
> 
> This is a good idea for the future `v0.3` API, but I'm not sure we need build it just for this feature.

Couldn't the new functionality use the builder, and the old function be reimplemented in terms of it internally?

Reply:
The latest revision with `ImportLibraryGenerator` looks really nice, thanks everyone.